source/helpers/test/Objects.test.tsx

```diff
@@ -48,3 +53,18 @@ describe("Object helper functions", () => {
     expect(firstFourth).toEqual(false);
   });
 });
+
+describe("Deep copy", () => {
+  const data = { hello: "world" };
```

Comment:
I think a nested object here would be great, otherwise it's just a test for shallow copy and not deep copy.

Reply:
Fixed!